Replace launchtemplate and launchtask in battle anims

ghoulslash opened the pull request:
Adding python script to replace `launchtemplate` with `createsprite`. Same with launchtask and createvisualtask

ANIM_PAL uses have also been removed in favor the pret's F_PAL_xx macros

SBird1337 reviewed (dismissed):
if the script is a one time thing we should probably not have it in the upstream tree.

ghoulslash commented:
> if the script is a one time thing we should probably not have it in the upstream tree.

removed script. also replaced launchtask with createvisualtask and fixed conflicts. ready for review

ghoulslash commented on asm/macros/battle_anim_script.inc:
```diff
@@ -289,39 +289,7 @@
-	.endm
-
+@ keep CFRU macros cause I'm lazy
 .macro setblends setblends_value
```
> Why not get rid of the setblends and launchsoundtask macros as well?

done
